### PR TITLE
INFODEV-3622

### PR DESCRIPTION
**Pull request by WilliamLoy**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by WilliamLoy** [comment]
@cyrichardson got this one ready for you when you have availability

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-dfb9d1d90a/how-to/)

**Comment by cyrichardson** [comment]
@WilliamLoy Suggestions here: 
[set-up-an-api-key-cloud-office-control-panel_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2912993/set-up-an-api-key-cloud-office-control-panel_cyrEdits.docx)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-5b6ad4f32f/how-to/). :bow:

File: content/rackspace-email/set-up-an-api-key-cloud-office-control-panel.md [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
 
-The Rackspace Email API provides most of the functions of the control
-panel through a REST-based web API. Whether you are adding a new
-customer account (for resellers), adding mailboxes, or using any of the
-other supported features, the API allows your application to administer
-the changes. These changes can be applied independent of your
-application's language or nature. Documentation for the API is located
-at:
+The Rackspace Email application programming interface (API) provides most of the functions of the control
+panel through a REST-based web API. The API allows you to programmatically administer common tasks by using your own application to perform common tasks such as adding mailboxes and more. You can apply these changes can be applied independent of your
```
Comment (stephamon) [inline content/rackspace-email/set-up-an-api-key-cloud-office-control-panel.md]: You can apply these changes independent of your applications's language or nature.

OR

These changes can be applied independent of your application's language or nature.

Not both :)

**Comment by cyrichardson** [comment]
@WilliamLoy @stephamon I did a quick commit to make the change.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-8ea349d9ec/how-to/)

**Pull request merged**